shared/models/videos/video-privacy.enum.ts

```diff
@@ -1,4 +1,5 @@
 export const enum VideoPrivacy {
+  SCHEDULED = -1,
```

Comment:
This is not a video privacy, then should not be added to this enum. It's the reason why we used `VideoEdit.SPECIAL_SCHEDULED_PRIVACY`

Reply:
True, but the form field name is "privacy". Can we rename the enum to `VideoPrivacyOption` or `PublishOption`?

Reply:
This enum? No because it's used by the server, that only accept these values. But you can create another enum in the client named `PublishOption`.